Github actions is failing on the percona role

Percona role seems to be failing github actions:

```
TASK [perconaxdb : Install packages] *******************************************
  Error: "cache_updated": false, "changed": false, "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"      install 'percona-xtradb-cluster-57' 'python3-mysqldb' 'python-mysqldb' 'python-dev'' failed: E: Sub-process /usr/bin/dpkg returned an error code (1)\n", "rc": 100, "stderr": "E: Sub-process /usr/bin/dpkg returned an error code (1)\n", "stderr_lines": ["E: Sub-process /usr/bin/dpkg returned an error code (1)"], "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nThe following additional packages will be installed:\n  curl debsums libaio1 libcurl4 libcurl4-openssl-dev libdbd-mysql-perl\n  libdbi-perl libev4 libfile-fnmatch-perl libmecab2 libnuma1 libpython-dev\n  libpython2.7-dev lsof percona-xtrabackup-24\n  percona-xtradb-cluster-client-5.7 percona-xtradb-cluster-common-5.7\n  percona-xtradb-cluster-server-5.7 python2.7-dev qpress rsync socat\nSuggested packages:\n  libcurl4-doc libidn11-dev libkrb5-dev libldap2-dev librtmp-dev libssh2-1-dev\n  libclone-perl libmldbm-perl libnet-daemon-perl libsql-statement-perl tinyca\n  netcat-openbsd pv python-egenix-mxdatetime python-mysqldb-dbg\n  python3-mysqldb-dbg\nThe following NEW packages will be installed:\n  debsums libaio1 libcurl4-openssl-dev libdbd-mysql-perl libdbi-perl libev4\n  libfile-fnmatch-perl libmecab2 libnuma1 libpython-dev libpython2.7-dev lsof\n  percona-xtrabackup-24 percona-xtradb-cluster-57\n  percona-xtradb-cluster-client-5.7 percona-xtradb-cluster-common-5.7\n  percona-xtradb-cluster-server-5.7 python-dev python-mysqldb python2.7-dev\n  python3-mysqldb qpress rsync socat\nThe following packages will be upgraded:\n  curl libcurl4\n2 upgraded, 24 newly installed, 0 to remove and 166 not upgraded.\nNeed to get 53.3 MB of archives.\nAfter this operation, 234 MB of additional disk space will be used.\nGet:1 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-common-5.7 amd64 5.7.33-31.49-1.bionic [27.7 kB]\nGet:2 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libfile-fnmatch-perl amd64 0.02-2build4 [9986 B]\nGet:3 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-client-5.7 amd64 5.7.33-31.49-1.bionic [2436 kB]\nGet:4 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtrabackup-24 amd64 2.4.22-1.bionic [6617 kB]\nGet:5 http://archive.ubuntu.com/ubuntu bionic/universe amd64 debsums all 2.2.2 [42.3 kB]\nGet:6 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdbi-perl amd64 1.640-1ubuntu0.2 [724 kB]\nGet:7 http://repo.percona.com/percona/apt bionic/main amd64 qpress amd64 11-1.bionic [33.4 kB]\nGet:8 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-server-5.7 amd64 5.7.33-31.49-1.bionic [12.8 MB]\nGet:9 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-57 amd64 5.7.33-31.49-1.bionic [5320 B]\nGet:10 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libdbd-mysql-perl amd64 4.046-1 [82.0 kB]\nGet:11 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libaio1 amd64 0.3.110-5ubuntu0.1 [6476 B]\nGet:12 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 curl amd64 7.58.0-2ubuntu3.12 [159 kB]\nGet:13 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcurl4 amd64 7.58.0-2ubuntu3.12 [214 kB]\nGet:14 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libmecab2 amd64 0.996-5 [257 kB]\nGet:15 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libnuma1 amd64 2.0.11-2.1ubuntu0.1 [22.0 kB]\nGet:16 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcurl4-openssl-dev amd64 7.58.0-2ubuntu3.12 [294 kB]\nGet:17 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 rsync amd64 3.1.2-2.1ubuntu1.1 [334 kB]\nGet:18 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libev4 amd64 1:4.22-1 [26.3 kB]\nGet:19 http://archive.ubuntu.com/ubuntu bionic/main amd64 socat amd64 1.7.3.2-2ubuntu2 [342 kB]\nGet:20 http://archive.ubuntu.com/ubuntu bionic/main amd64 lsof amd64 4.89+dfsg-0.1 [248 kB]\nGet:21 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-dev amd64 2.7.17-1~18.04ubuntu1.6 [28.3 MB]\nGet:22 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-dev amd64 2.7.15~rc1-1 [7684 B]\nGet:23 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-dev amd64 2.7.17-1~18.04ubuntu1.6 [279 kB]\nGet:24 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-dev amd64 2.7.15~rc1-1 [1256 B]\nGet:25 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-mysqldb amd64 1.3.10-1build1 [49.2 kB]\nGet:26 http://archive.ubuntu.com/ubuntu bionic/main amd64 python3-mysqldb amd64 1.3.10-1build1 [46.0 kB]\nPreconfiguring packages ...\nFetched 53.3 MB in 2s (35.0 MB/s)\nSelecting previously unselected package libfile-fnmatch-perl.\r\n(Reading database ... \r(Reading database ... 5%\r(Reading database ... 10%\r(Reading database ... 15%\r(Reading database ... 20%\r(Reading database ... 25%\r(Reading database ... 30%\r(Reading database ... 35%\r(Reading database ... 40%\r(Reading database ... 45%\r(Reading database ... 50%\r(Reading database ... 55%\r(Reading database ... 60%\r(Reading database ... 65%\r(Reading database ... 70%\r(Reading database ... 75%\r(Reading database ... 80%\r(Reading database ... 85%\r(Reading database ... 90%\r(Reading database ... 95%\r(Reading database ... 100%\r(Reading database ... 24855 files and directories currently installed.)\r\nPreparing to unpack .../libfile-fnmatch-perl_0.02-2build4_amd64.deb ...\r\nUnpacking libfile-fnmatch-perl (0.02-2build4) ...\r\nSelecting previously unselected package debsums.\r\nPreparing to unpack .../archives/debsums_2.2.2_all.deb ...\r\nUnpacking debsums (2.2.2) ...\r\nSetting up libfile-fnmatch-perl (0.02-2build4) ...\r\nSetting up debsums (2.2.2) ...\r\nSelecting previously unselected package percona-xtradb-cluster-common-5.7.\r\n(Reading database ... \r(Reading database ... 5%\r(Reading database ... 10%\r(Reading database ... 15%\r(Reading database ... 20%\r(Reading database ... 25%\r(Reading database ... 30%\r(Reading database ... 35%\r(Reading database ... 40%\r(Reading database ... 45%\r(Reading database ... 50%\r(Reading database ... 55%\r(Reading database ... 60%\r(Reading database ... 65%\r(Reading database ... 70%\r(Reading database ... 75%\r(Reading database ... 80%\r(Reading database ... 85%\r(Reading database ... 90%\r(Reading database ... 95%\r(Reading database ... 100%\r(Reading database ... 24895 files and directories currently installed.)\r\nPreparing to unpack .../00-percona-xtradb-cluster-common-5.7_5.7.33-31.49-1.bionic_amd64.deb ...\r\nUnpacking percona-xtradb-cluster-common-5.7 (5.7.33-31.49-1.bionic) ...\r\nSelecting previously unselected package libdbi-perl.\r\nPreparing to unpack .../01-libdbi-perl_1.640-1ubuntu0.2_amd64.deb ...\r\nUnpacking libdbi-perl (1.640-1ubuntu0.2) ...\r\nSelecting previously unselected package percona-xtradb-cluster-client-5.7.\r\nPreparing to unpack .../02-percona-xtradb-cluster-client-5.7_5.7.33-31.49-1.bionic_amd64.deb ...\r\nUnpacking percona-xtradb-cluster-client-5.7 (5.7.33-31.49-1.bionic) ...\r\nSelecting previously unselected package libdbd-mysql-perl.\r\nPreparing to unpack .../03-libdbd-mysql-perl_4.046-1_amd64.deb ...\r\nUnpacking libdbd-mysql-perl (4.046-1) ...\r\nSelecting previously unselected package libaio1:amd64.\r\nPreparing to unpack .../04-libaio1_0.3.110-5ubuntu0.1_amd64.deb ...\r\nUnpacking libaio1:amd64 (0.3.110-5ubuntu0.1) ...\r\nPreparing to unpack .../05-curl_7.58.0-2ubuntu3.12_amd64.deb ...\r\nUnpacking curl (7.58.0-2ubuntu3.12) over (7.58.0-2ubuntu3.2) ...\r\nPreparing to unpack .../06-libcurl4_7.58.0-2ubuntu3.12_amd64.deb ...\r\nUnpacking libcurl4:amd64 (7.58.0-2ubuntu3.12) over (7.58.0-2ubuntu3.2) ...\r\nSelecting previously unselected package libmecab2:amd64.\r\nPreparing to unpack .../07-libmecab2_0.996-5_amd64.deb ...\r\nUnpacking libmecab2:amd64 (0.996-5) ...\r\nSelecting previously unselected package libnuma1:amd64.\r\nPreparing to unpack .../08-libnuma1_2.0.11-2.1ubuntu0.1_amd64.deb ...\r\nUnpacking libnuma1:amd64 (2.0.11-2.1ubuntu0.1) ...\r\nSelecting previously unselected package libcurl4-openssl-dev:amd64.\r\nPreparing to unpack .../09-libcurl4-openssl-dev_7.58.0-2ubuntu3.12_amd64.deb ...\r\nUnpacking libcurl4-openssl-dev:amd64 (7.58.0-2ubuntu3.12) ...\r\nSelecting previously unselected package rsync.\r\nPreparing to unpack .../10-rsync_3.1.2-2.1ubuntu1.1_amd64.deb ...\r\nUnpacking rsync (3.1.2-2.1ubuntu1.1) ...\r\nSelecting previously unselected package libev4.\r\nPreparing to unpack .../11-libev4_1%3a4.22-1_amd64.deb ...\r\nUnpacking libev4 (1:4.22-1) ...\r\nSelecting previously unselected package percona-xtrabackup-24.\r\nPreparing to unpack .../12-percona-xtrabackup-24_2.4.22-1.bionic_amd64.deb ...\r\nUnpacking percona-xtrabackup-24 (2.4.22-1.bionic) ...\r\nSelecting previously unselected package socat.\r\nPreparing to unpack .../13-socat_1.7.3.2-2ubuntu2_amd64.deb ...\r\nUnpacking socat (1.7.3.2-2ubuntu2) ...\r\nSelecting previously unselected package lsof.\r\nPreparing to unpack .../14-lsof_4.89+dfsg-0.1_amd64.deb ...\r\nUnpacking lsof (4.89+dfsg-0.1) ...\r\nSelecting previously unselected package qpress.\r\nPreparing to unpack .../15-qpress_11-1.bionic_amd64.deb ...\r\nUnpacking qpress (11-1.bionic) ...\r\nSelecting previously unselected package percona-xtradb-cluster-server-5.7.\r\nPreparing to unpack .../16-percona-xtradb-cluster-server-5.7_5.7.33-31.49-1.bionic_amd64.deb ...\r\nUnpacking percona-xtradb-cluster-server-5.7 (5.7.33-31.49-1.bionic) ...\r\nSelecting previously unselected package libpython2.7-dev:amd64.\r\nPreparing to unpack .../17-libpython2.7-dev_2.7.17-1~18.04ubuntu1.6_amd64.deb ...\r\nUnpacking libpython2.7-dev:amd64 (2.7.17-1~18.04ubuntu1.6) ...\r\nSelecting previously unselected package libpython-dev:amd64.\r\nPreparing to unpack .../18-libpython-dev_2.7.15~rc1-1_amd64.deb ...\r\nUnpacking libpython-dev:amd64 (2.7.15~rc1-1) ...\r\nSelecting previously unselected package python2.7-dev.\r\nPreparing to unpack .../19-python2.7-dev_2.7.17-1~18.04ubuntu1.6_amd64.deb ...\r\nUnpacking python2.7-dev (2.7.17-1~18.04ubuntu1.6) ...\r\nSelecting previously unselected package python-dev.\r\nPreparing to unpack .../20-python-dev_2.7.15~rc1-1_amd64.deb ...\r\nUnpacking python-dev (2.7.15~rc1-1) ...\r\nSelecting previously unselected package python-mysqldb.\r\nPreparing to unpack .../21-python-mysqldb_1.3.10-1build1_amd64.deb ...\r\nUnpacking python-mysqldb (1.3.10-1build1) ...\r\nSelecting previously unselected package python3-mysqldb.\r\nPreparing to unpack .../22-python3-mysqldb_1.3.10-1build1_amd64.deb ...\r\nUnpacking python3-mysqldb (1.3.10-1build1) ...\r\nSelecting previously unselected package percona-xtradb-cluster-57.\r\nPreparing to unpack .../23-percona-xtradb-cluster-57_5.7.33-31.49-1.bionic_amd64.deb ...\r\nUnpacking percona-xtradb-cluster-57 (5.7.33-31.49-1.bionic) ...\r\nSetting up libev4 (1:4.22-1) ...\r\nSetting up qpress (11-1.bionic) ...\r\nSetting up socat (1.7.3.2-2ubuntu2) ...\r\nSetting up lsof (4.89+dfsg-0.1) ...\r\nSetting up libnuma1:amd64 (2.0.11-2.1ubuntu0.1) ...\r\nSetting up libcurl4:amd64 (7.58.0-2ubuntu3.12) ...\r\nSetting up libmecab2:amd64 (0.996-5) ...\r\nSetting up python3-mysqldb (1.3.10-1build1) ...\r\nSetting up rsync (3.1.2-2.1ubuntu1.1) ...\r\ninvoke-rc.d: could not determine current runlevel\r\ninvoke-rc.d: policy-rc.d denied execution of restart.\r\nCreated symlink /etc/systemd/system/multi-user.target.wants/rsync.service -> /lib/systemd/system/rsync.service.\r\nProcessing triggers for libc-bin (2.27-3ubuntu1) ...\r\nSetting up libaio1:amd64 (0.3.110-5ubuntu0.1) ...\r\nSetting up python-mysqldb (1.3.10-1build1) ...\r\nProcessing triggers for systemd (237-3ubuntu10.24) ...\r\nSetting up libpython2.7-dev:amd64 (2.7.17-1~18.04ubuntu1.6) ...\r\nSetting up percona-xtradb-cluster-common-5.7 (5.7.33-31.49-1.bionic) ...\r\ndebsums: missing file /usr/share/doc/mysql-common/copyright (from mysql-common package)\r\ndebsums: missing file /usr/share/doc/mysql-common/copyright (from mysql-common package)\r\ndebsums: missing file /usr/share/doc/mysql-common/copyright (from mysql-common package)\r\nupdate-alternatives: using /etc/mysql/percona-xtradb-cluster.cnf to provide /etc/mysql/my.cnf (my.cnf) in auto mode\r\nSetting up python2.7-dev (2.7.17-1~18.04ubuntu1.6) ...\r\nSetting up libcurl4-openssl-dev:amd64 (7.58.0-2ubuntu3.12) ...\r\nSetting up libdbi-perl (1.640-1ubuntu0.2) ...\r\nSetting up libpython-dev:amd64 (2.7.15~rc1-1) ...\r\nSetting up curl (7.58.0-2ubuntu3.12) ...\r\nSetting up python-dev (2.7.15~rc1-1) ...\r\nSetting up percona-xtradb-cluster-client-5.7 (5.7.33-31.49-1.bionic) ...\r\nSetting up libdbd-mysql-perl (4.046-1) ...\r\nSetting up percona-xtrabackup-24 (2.4.22-1.bionic) ...\r\nSetting up percona-xtradb-cluster-server-5.7 (5.7.33-31.49-1.bionic) ...\r\n2021-03-29T12:31:28.577097Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).\r\n2021-03-29T12:31:28.577154Z 0 [Warning] WSREP: Node is running in bootstrap/initialize mode. Disabling pxc_strict_mode checks\r\n2021-03-29T12:31:28.580201Z 0 [ERROR] Could not open file '/var/log/mysqld.log' for error logging: Permission denied\r\n2021-03-29T12:31:28.580221Z 0 [ERROR] Aborting\r\n\r\ndpkg: error processing package percona-xtradb-cluster-server-5.7 (--configure):\r\n installed percona-xtradb-cluster-server-5.7 package post-installation script subprocess returned error exit status 1\r\ndpkg: dependency problems prevent configuration of percona-xtradb-cluster-57:\r\n percona-xtradb-cluster-57 depends on percona-xtradb-cluster-server-5.7 (>= 5.7.33-31.49-1.bionic); however:\r\n  Package percona-xtradb-cluster-server-5.7 is not configured yet.\r\n\r\ndpkg: error processing package percona-xtradb-cluster-57 (--configure):\r\n dependency problems - leaving unconfigured\r\nProcessing triggers for libc-bin (2.27-3ubuntu1) ...\r\nProcessing triggers for systemd (237-3ubuntu10.24) ...\r\nErrors were encountered while processing:\r\n percona-xtradb-cluster-server-5.7\r\n percona-xtradb-cluster-57\r\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information...", "The following additional packages will be installed:", "  curl debsums libaio1 libcurl4 libcurl4-openssl-dev libdbd-mysql-perl", "  libdbi-perl libev4 libfile-fnmatch-perl libmecab2 libnuma1 libpython-dev", "  libpython2.7-dev lsof percona-xtrabackup-24", "  percona-xtradb-cluster-client-5.7 percona-xtradb-cluster-common-5.7", "  percona-xtradb-cluster-server-5.7 python2.7-dev qpress rsync socat", "Suggested packages:", "  libcurl4-doc libidn11-dev libkrb5-dev libldap2-dev librtmp-dev libssh2-1-dev", "  libclone-perl libmldbm-perl libnet-daemon-perl libsql-statement-perl tinyca", "  netcat-openbsd pv python-egenix-mxdatetime python-mysqldb-dbg", "  python3-mysqldb-dbg", "The following NEW packages will be installed:", "  debsums libaio1 libcurl4-openssl-dev libdbd-mysql-perl libdbi-perl libev4", "  libfile-fnmatch-perl libmecab2 libnuma1 libpython-dev libpython2.7-dev lsof", "  percona-xtrabackup-24 percona-xtradb-cluster-57", "  percona-xtradb-cluster-client-5.7 percona-xtradb-cluster-common-5.7", "  percona-xtradb-cluster-server-5.7 python-dev python-mysqldb python2.7-dev", "  python3-mysqldb qpress rsync socat", "The following packages will be upgraded:", "  curl libcurl4", "2 upgraded, 24 newly installed, 0 to remove and 166 not upgraded.", "Need to get 53.3 MB of archives.", "After this operation, 234 MB of additional disk space will be used.", "Get:1 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-common-5.7 amd64 5.7.33-31.49-1.bionic [27.7 kB]", "Get:2 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libfile-fnmatch-perl amd64 0.02-2build4 [9986 B]", "Get:3 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-client-5.7 amd64 5.7.33-31.49-1.bionic [2436 kB]", "Get:4 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtrabackup-24 amd64 2.4.22-1.bionic [6617 kB]", "Get:5 http://archive.ubuntu.com/ubuntu bionic/universe amd64 debsums all 2.2.2 [42.3 kB]", "Get:6 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdbi-perl amd64 1.640-1ubuntu0.2 [724 kB]", "Get:7 http://repo.percona.com/percona/apt bionic/main amd64 qpress amd64 11-1.bionic [33.4 kB]", "Get:8 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-server-5.7 amd64 5.7.33-31.49-1.bionic [12.8 MB]", "Get:9 http://repo.percona.com/percona/apt bionic/main amd64 percona-xtradb-cluster-57 amd64 5.7.33-31.49-1.bionic [5320 B]", "Get:10 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libdbd-mysql-perl amd64 4.046-1 [82.0 kB]", "Get:11 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libaio1 amd64 0.3.110-5ubuntu0.1 [6476 B]", "Get:12 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 curl amd64 7.58.0-2ubuntu3.12 [159 kB]", "Get:13 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcurl4 amd64 7.58.0-2ubuntu3.12 [214 kB]", "Get:14 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libmecab2 amd64 0.996-5 [257 kB]", "Get:15 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libnuma1 amd64 2.0.11-2.1ubuntu0.1 [22.0 kB]", "Get:16 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcurl4-openssl-dev amd64 7.58.0-2ubuntu3.12 [294 kB]", "Get:17 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 rsync amd64 3.1.2-2.1ubuntu1.1 [334 kB]", "Get:18 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libev4 amd64 1:4.22-1 [26.3 kB]", "Get:19 http://archive.ubuntu.com/ubuntu bionic/main amd64 socat amd64 1.7.3.2-2ubuntu2 [342 kB]", "Get:20 http://archive.ubuntu.com/ubuntu bionic/main amd64 lsof amd64 4.89+dfsg-0.1 [248 kB]", "Get:21 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-dev amd64 2.7.17-1~18.04ubuntu1.6 [28.3 MB]", "Get:22 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-dev amd64 2.7.15~rc1-1 [7684 B]", "Get:23 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-dev amd64 2.7.17-1~18.04ubuntu1.6 [279 kB]", "Get:24 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-dev amd64 2.7.15~rc1-1 [1256 B]", "Get:25 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-mysqldb amd64 1.3.10-1build1 [49.2 kB]", "Get:26 http://archive.ubuntu.com/ubuntu bionic/main amd64 python3-mysqldb amd64 1.3.10-1build1 [46.0 kB]", "Preconfiguring packages ...", "Fetched 53.3 MB in 2s (35.0 MB/s)", "Selecting previously unselected package libfile-fnmatch-perl.", "(Reading database ... ", "(Reading database ... 5%", "(Reading database ... 10%", "(Reading database ... 15%", "(Reading database ... 20%", "(Reading database ... 25%", "(Reading database ... 30%", "(Reading database ... 35%", "(Reading database ... 40%", "(Reading database ... 45%", "(Reading database ... 50%", "(Reading database ... 55%", "(Reading database ... 60%", "(Reading database ... 65%", "(Reading database ... 70%", "(Reading database ... 75%", "(Reading database ... 80%", "(Reading database ... 85%", "(Reading database ... 90%", "(Reading database ... 95%", "(Reading database ... 100%", "(Reading database ... 24855 files and directories currently installed.)", "Preparing to unpack .../libfile-fnmatch-perl_0.02-2build4_amd64.deb ...", "Unpacking libfile-fnmatch-perl (0.02-2build4) ...", "Selecting previously unselected package debsums.", "Preparing to unpack .../archives/debsums_2.2.2_all.deb ...", "Unpacking debsums (2.2.2) ...", "Setting up libfile-fnmatch-perl (0.02-2build4) ...", "Setting up debsums (2.2.2) ...", "Selecting previously unselected package percona-xtradb-cluster-common-5.7.", "(Reading database ... ", "(Reading database ... 5%", "(Reading database ... 10%", "(Reading database ... 15%", "(Reading database ... 20%", "(Reading database ... 25%", "(Reading database ... 30%", "(Reading database ... 35%", "(Reading database ... 40%", "(Reading database ... 45%", "(Reading database ... 50%", "(Reading database ... 55%", "(Reading database ... 60%", "(Reading database ... 65%", "(Reading database ... 70%", "(Reading database ... 75%", "(Reading database ... 80%", "(Reading database ... 85%", "(Reading database ... 90%", "(Reading database ... 95%", "(Reading database ... 100%", "(Reading database ... 24895 files and directories currently installed.)", "Preparing to unpack .../00-percona-xtradb-cluster-common-5.7_5.7.33-31.49-1.bionic_amd64.deb ...", "Unpacking percona-xtradb-cluster-common-5.7 (5.7.33-31.49-1.bionic) ...", "Selecting previously unselected package libdbi-perl.", "Preparing to unpack .../01-libdbi-perl_1.640-1ubuntu0.2_amd64.deb ...", "Unpacking libdbi-perl (1.640-1ubuntu0.2) ...", "Selecting previously unselected package percona-xtradb-cluster-client-5.7.", "Preparing to unpack .../02-percona-xtradb-cluster-client-5.7_5.7.33-31.49-1.bionic_amd64.deb ...", "Unpacking percona-xtradb-cluster-client-5.7 (5.7.33-31.49-1.bionic) ...", "Selecting previously unselected package libdbd-mysql-perl.", "Preparing to unpack .../03-libdbd-mysql-perl_4.046-1_amd64.deb ...", "Unpacking libdbd-mysql-perl (4.046-1) ...", "Selecting previously unselected package libaio1:amd64.", "Preparing to unpack .../04-libaio1_0.3.110-5ubuntu0.1_amd64.deb ...", "Unpacking libaio1:amd64 (0.3.110-5ubuntu0.1) ...", "Preparing to unpack .../05-curl_7.58.0-2ubuntu3.12_amd64.deb ...", "Unpacking curl (7.58.0-2ubuntu3.12) over (7.58.0-2ubuntu3.2) ...", "Preparing to unpack .../06-libcurl4_7.58.0-2ubuntu3.12_amd64.deb ...", "Unpacking libcurl4:amd64 (7.58.0-2ubuntu3.12) over (7.58.0-2ubuntu3.2) ...", "Selecting previously unselected package libmecab2:amd64.", "Preparing to unpack .../07-libmecab2_0.996-5_amd64.deb ...", "Unpacking libmecab2:amd64 (0.996-5) ...", "Selecting previously unselected package libnuma1:amd64.", "Preparing to unpack .../08-libnuma1_2.0.11-2.1ubuntu0.1_amd64.deb ...", "Unpacking libnuma1:amd64 (2.0.11-2.1ubuntu0.1) ...", "Selecting previously unselected package libcurl4-openssl-dev:amd64.", "Preparing to unpack .../09-libcurl4-openssl-dev_7.58.0-2ubuntu3.12_amd64.deb ...", "Unpacking libcurl4-openssl-dev:amd64 (7.58.0-2ubuntu3.12) ...", "Selecting previously unselected package rsync.", "Preparing to unpack .../10-rsync_3.1.2-2.1ubuntu1.1_amd64.deb ...", "Unpacking rsync (3.1.2-2.1ubuntu1.1) ...", "Selecting previously unselected package libev4.", "Preparing to unpack .../11-libev4_1%3a4.22-1_amd64.deb ...", "Unpacking libev4 (1:4.22-1) ...", "Selecting previously unselected package percona-xtrabackup-24.", "Preparing to unpack .../12-percona-xtrabackup-24_2.4.22-1.bionic_amd64.deb ...", "Unpacking percona-xtrabackup-24 (2.4.22-1.bionic) ...", "Selecting previously unselected package socat.", "Preparing to unpack .../13-socat_1.7.3.2-2ubuntu2_amd64.deb ...", "Unpacking socat (1.7.3.2-2ubuntu2) ...", "Selecting previously unselected package lsof.", "Preparing to unpack .../14-lsof_4.89+dfsg-0.1_amd64.deb ...", "Unpacking lsof (4.89+dfsg-0.1) ...", "Selecting previously unselected package qpress.", "Preparing to unpack .../15-qpress_11-1.bionic_amd64.deb ...", "Unpacking qpress (11-1.bionic) ...", "Selecting previously unselected package percona-xtradb-cluster-server-5.7.", "Preparing to unpack .../16-percona-xtradb-cluster-server-5.7_5.7.33-31.49-1.bionic_amd64.deb ...", "Unpacking percona-xtradb-cluster-server-5.7 (5.7.33-31.49-1.bionic) ...", "Selecting previously unselected package libpython2.7-dev:amd64.", "Preparing to unpack .../17-libpython2.7-dev_2.7.17-1~18.04ubuntu1.6_amd64.deb ...", "Unpacking libpython2.7-dev:amd64 (2.7.17-1~18.04ubuntu1.6) ...", "Selecting previously unselected package libpython-dev:amd64.", "Preparing to unpack .../18-libpython-dev_2.7.15~rc1-1_amd64.deb ...", "Unpacking libpython-dev:amd64 (2.7.15~rc1-1) ...", "Selecting previously unselected package python2.7-dev.", "Preparing to unpack .../19-python2.7-dev_2.7.17-1~18.04ubuntu1.6_amd64.deb ...", "Unpacking python2.7-dev (2.7.17-1~18.04ubuntu1.6) ...", "Selecting previously unselected package python-dev.", "Preparing to unpack .../20-python-dev_2.7.15~rc1-1_amd64.deb ...", "Unpacking python-dev (2.7.15~rc1-1) ...", "Selecting previously unselected package python-mysqldb.", "Preparing to unpack .../21-python-mysqldb_1.3.10-1build1_amd64.deb ...", "Unpacking python-mysqldb (1.3.10-1build1) ...", "Selecting previously unselected package python3-mysqldb.", "Preparing to unpack .../22-python3-mysqldb_1.3.10-1build1_amd64.deb ...", "Unpacking python3-mysqldb (1.3.10-1build1) ...", "Selecting previously unselected package percona-xtradb-cluster-57.", "Preparing to unpack .../23-percona-xtradb-cluster-57_5.7.33-31.49-1.bionic_amd64.deb ...", "Unpacking percona-xtradb-cluster-57 (5.7.33-31.49-1.bionic) ...", "Setting up libev4 (1:4.22-1) ...", "Setting up qpress (11-1.bionic) ...", "Setting up socat (1.7.3.2-2ubuntu2) ...", "Setting up lsof (4.89+dfsg-0.1) ...", "Setting up libnuma1:amd64 (2.0.11-2.1ubuntu0.1) ...", "Setting up libcurl4:amd64 (7.58.0-2ubuntu3.12) ...", "Setting up libmecab2:amd64 (0.996-5) ...", "Setting up python3-mysqldb (1.3.10-1build1) ...", "Setting up rsync (3.1.2-2.1ubuntu1.1) ...", "invoke-rc.d: could not determine current runlevel", "invoke-rc.d: policy-rc.d denied execution of restart.", "Created symlink /etc/systemd/system/multi-user.target.wants/rsync.service -> /lib/systemd/system/rsync.service.", "Processing triggers for libc-bin (2.27-3ubuntu1) ...", "Setting up libaio1:amd64 (0.3.110-5ubuntu0.1) ...", "Setting up python-mysqldb (1.3.10-1build1) ...", "Processing triggers for systemd (237-3ubuntu10.24) ...", "Setting up libpython2.7-dev:amd64 (2.7.17-1~18.04ubuntu1.6) ...", "Setting up percona-xtradb-cluster-common-5.7 (5.7.33-31.49-1.bionic) ...", "debsums: missing file /usr/share/doc/mysql-common/copyright (from mysql-common package)", "debsums: missing file /usr/share/doc/mysql-common/copyright (from mysql-common package)", "debsums: missing file /usr/share/doc/mysql-common/copyright (from mysql-common package)", "update-alternatives: using /etc/mysql/percona-xtradb-cluster.cnf to provide /etc/mysql/my.cnf (my.cnf) in auto mode", "Setting up python2.7-dev (2.7.17-1~18.04ubuntu1.6) ...", "Setting up libcurl4-openssl-dev:amd64 (7.58.0-2ubuntu3.12) ...", "Setting up libdbi-perl (1.640-1ubuntu0.2) ...", "Setting up libpython-dev:amd64 (2.7.15~rc1-1) ...", "Setting up curl (7.58.0-2ubuntu3.12) ...", "Setting up python-dev (2.7.15~rc1-1) ...", "Setting up percona-xtradb-cluster-client-5.7 (5.7.33-31.49-1.bionic) ...", "Setting up libdbd-mysql-perl (4.046-1) ...", "Setting up percona-xtrabackup-24 (2.4.22-1.bionic) ...", "Setting up percona-xtradb-cluster-server-5.7 (5.7.33-31.49-1.bionic) ...", "2021-03-29T12:31:28.577097Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).", "2021-03-29T12:31:28.577154Z 0 [Warning] WSREP: Node is running in bootstrap/initialize mode. Disabling pxc_strict_mode checks", "2021-03-29T12:31:28.580201Z 0 [ERROR] Could not open file '/var/log/mysqld.log' for error logging: Permission denied", "2021-03-29T12:31:28.580221Z 0 [ERROR] Aborting", "", "dpkg: error processing package percona-xtradb-cluster-server-5.7 (--configure):", " installed percona-xtradb-cluster-server-5.7 package post-installation script subprocess returned error exit status 1", "dpkg: dependency problems prevent configuration of percona-xtradb-cluster-57:", " percona-xtradb-cluster-57 depends on percona-xtradb-cluster-server-5.7 (>= 5.7.33-31.49-1.bionic); however:", "  Package percona-xtradb-cluster-server-5.7 is not configured yet.", "", "dpkg: error processing package percona-xtradb-cluster-57 (--configure):", " dependency problems - leaving unconfigured", "Processing triggers for libc-bin (2.27-3ubuntu1) ...", "Processing triggers for systemd (237-3ubuntu10.24) ...", "Errors were encountered while processing:", " percona-xtradb-cluster-server-5.7", " percona-xtradb-cluster-57"]}
```

--- a/roles/perconaxdb/defaults/main.yml
+++ b/roles/perconaxdb/defaults/main.yml
@@ -64,6 +64,7 @@ xtradb_leader_node: "{{ hostvars[ groups[xtradb_nodes_group][0] ].ansible_defaul
 
 # Defines the version
 xtradb_version: "57"
+xtradb_server_version: "5.7"
 
 # Defines the service systemd
 xtradb_service: mysql

--- a/roles/perconaxdb/vars/main.yml
+++ b/roles/perconaxdb/vars/main.yml
@@ -6,6 +6,7 @@ xtradb_confdir: /etc/mysql/percona-xtradb-cluster.conf.d
 xtradb_config_file: "{{ xtradb_confdir }}/wsrep.cnf"
 xtradb_packages:
   - "percona-xtradb-cluster-{{ xtradb_version }}"
+  - "percona-xtradb-cluster-server-{{ xtradb_server_version }}"
   - python3-mysqldb
   - python-mysqldb
   - python-dev

